tests/test_bot_config.py

```
@@ -71,6 +71,152 @@ def test_bot_non_dict_config(self):
+
+        long_text = "This is a very long Bible verse that should be split into multiple messages because it exceeds the split message length"
+        with patch(
+            "biblebot.bot.get_bible_text",
+            new=AsyncMock(return_value=(long_text, "John 3:16")),
+        ):
+            await bot.handle_scripture_command(
+                "!test:example.org", "John 3:16", "kjv", mock_event
+            )
+
+            # Should send reaction + multiple messages
+            assert mock_client.room_send.call_count > 2
```

Ruff (0.12.2)

Use of `assert` detected

(S101)